### PR TITLE
Kubernetes provider 1.2.0 release notes

### PR DESCRIPTION
**Pull request by linglingye001**

### CODE REVIEW

File: releaseNotes/KubernetesProvider.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
 
 [Image][image] | [Sample][sample]
 
+## 1.2.0 - 
+
+### New Features
+* Added support for feature flags, which allows feature flag settings to be saved to a Kubernetes ConfigMap.
+    * Added the `featureFlag.selectors` property for feature flag filtering.
```
Comment (RichardChen820) [inline releaseNotes/KubernetesProvider.md:9]: Added support for feature flags, which allows projecting feature flags to the target ConfigMap.
  * Added the `spec.featureFlag.selectors` property for filtering the feature flags by multiple combinations of `keyFilter` and `labelFilter` .
  * Added the `sepc.featureFlag.refresh` property for periodically updating feature flags with the latest data from App Configuration.

File: releaseNotes/KubernetesProvider.md [hunk 2]
```diff
@@ -2,6 +2,17 @@
...
 
+## 1.2.0 - 
+
+### New Features
+* Added support for feature flags, which allows feature flag settings to be saved to a Kubernetes ConfigMap.
+    * Added the `featureFlag.selectors` property for feature flag filtering.
+    * Added the `featureFlag.refresh` property for periodically fetching latest feature flag values and update Kubernetes ConfigMap accordingly.
+
+### Enhancements
+* Added support for setting `nodeSelector`, `affinity` and `tolerations` when use helm to install Azure App Configuration Kubernetes Provider. It's useful to control which nodes the Kubernetes Provider pod can be scheduled. [#858](https://github.com/Azure/AppConfiguration/issues/858)
```
Comment (RichardChen820) [inline releaseNotes/KubernetesProvider.md:13]: Added support for setting `nodeSelector`, `affinity` and `tolerations` when use helm to install Azure App Configuration Kubernetes Provider. It's useful for node assignment management

File: releaseNotes/KubernetesProvider.md [hunk 3]
```diff
@@ -2,6 +2,17 @@
...
+* Added support for setting `autoscaling.enabled` when use helm to install Azure App Configuration Kubernetes Provider. It defaults to `false` and must be set to `true` for enabling [HorizontalPodAutoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/) to automatically manage scale for Kubernetes Provider.
```
Comment (RichardChen820) [inline releaseNotes/KubernetesProvider.md:14]: Added support for setting `autoscaling` when using helm to install Azure App Configuration Kubernetes Provider. Setting `autoscaling.enabled` to `true` to use [HorizontalPodAutoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/) for managing pod autoscaling of Kubernetes Provider.

File: releaseNotes/KubernetesProvider.md [hunk 4]
```diff
@@ -2,6 +2,17 @@
 
 [Image][image] | [Sample][sample]
 
+## 1.2.0 - 
+
+### New Features
+* Added support for feature flags, which allows feature flag settings to be saved to a Kubernetes ConfigMap.
```
Comment (zhenlan) [inline releaseNotes/KubernetesProvider.md:8]: This depends on the release of FM libraries of other languages. We will need some coordination before we can declare the support for feature flags.

File: releaseNotes/KubernetesProvider.md [hunk 5]
```diff
@@ -2,6 +2,17 @@
 
 [Image][image] | [Sample][sample]
 
+## 1.2.0 - 
+
+### New Features
```
Comment (zhenlan) [inline releaseNotes/KubernetesProvider.md:7]: New features are enhancement too.

File: releaseNotes/KubernetesProvider.md (hunk 1 above)
Comment (zhenlan) [reply releaseNotes/KubernetesProvider.md:9]: This is a bit too many details. We can point people to the documentation. The release notes aren't documentation.

File: releaseNotes/KubernetesProvider.md (hunk 4 above)
Comment (linglingye001) [reply releaseNotes/KubernetesProvider.md:8]: I see. So the release sequence should be FM libraries (dotnet/java spring) -> k8s provider.

Comment (zhenlan) [reply releaseNotes/KubernetesProvider.md]: Yes, something like that, and in k8s provider release, we will say which version of FM library people need to use and on which platform. We will need quickstart/tutorial docs for k8s provider FM as well.

File: releaseNotes/KubernetesProvider.md [hunk 6]
```diff
@@ -6,8 +6,8 @@
 
 ### Enhancements
 * Added support for feature flags, which allows projecting feature flags to the target ConfigMap using [Microsoft Feature Flag schema](https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureFlag.v1.1.0.schema.json). The ConfigMap can be consumed by Feature Management libraries in all languages.
-    * [.NET Feature Management](https://github.com/microsoft/FeatureManagement-Dotnet) v3.2.0
-    * [Spring Cloud Azure Feature Management](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/spring/spring-cloud-azure-feature-management) v5.9.0-beta.1
+    * [.NET Feature Management](https://github.com/microsoft/FeatureManagement-Dotnet) (version 3.2.0 or later)
```
Comment (RichardChen820) [inline releaseNotes/KubernetesProvider.md:9]: v3.2.0 or later ?

File: releaseNotes/KubernetesProvider.md [hunk 7]
```diff
@@ -2,6 +2,15 @@
 
 [Image][image] | [Sample][sample]
 
+## 1.2.0 - 
+
+### Enhancements
+* Added support for feature flags, which allows projecting feature flags to the target ConfigMap using [Microsoft Feature Flag schema](https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureManagement.v1.0.0.schema.json). The ConfigMap can be consumed by Feature Management libraries in all languages.
```
Comment (zhenlan) [inline releaseNotes/KubernetesProvider.md:8]: This is not something that can be explained in a sentence or two. The support in other languages is indepent on this release. In this case, I would skip most of the details and have something like below. We can include a link to a document if we have one ready when this release note is published.

* Added support for downloading feature flags from Azure App Configuration.

Comment (linglingye001) [reply releaseNotes/KubernetesProvider.md:8]: Not sure the "link to a doc" refers to what doc, and where the link to include.

Comment (zhenlan) [reply releaseNotes/KubernetesProvider.md]: The quickstart for feature management using the k8s provider.

**Review by zhenlan — APPROVED** [review]

**Pull request merged**